test/dotnet-publish.Tests/GivenDotnetPublishPublishesProjects.cs

```diff
@@ -152,13 +152,12 @@ public void ItPublishesFrameworkDependentWithoutRid(string args)
             var testAppName = "MSBuildTestApp";
             var outputDirectory = PublishApp(testAppName, rid: null, args: args);
 
-            outputDirectory.Should().OnlyHaveFiles(new[] {
-                $"{testAppName}{Constants.ExeSuffix}",
+            outputDirectory.Should().OnlyHaveFiles(AssertionHelper.AppendApphostOnNonMacOS(testAppName, new[] {
```

Comment:
Could you make AppendApphostOnMacOS an extension method to the assertion and write it like this:

outputDirectory.Should().OnlyHaveFiles(...).And.AppendApphostOnNonMacOS()?

The way it is now reads strangely.

Reply:
Talked in person. The existing is fine since it will fail at outputDirectory.Should()**.OnlyHaveFiles(...)**.And.AppendApphostOnNonMacOS()?